src/org/labkey/test/components/dataset/AdvancedDatasetSettingsDialog.java

```diff
@@ -0,0 +1,142 @@
+package org.labkey.test.components.dataset;
```

Comment:
Packages should be named based on the module they are related to or the JavaScript component they wrap. Something like:
```suggestion
package org.labkey.test.components.study;
```
_(but don't put it in the platform repository)_

Reply:
moved it to the existing study package there